manifests/p/PrismLauncher/PrismLauncher/6.3/PrismLauncher.PrismLauncher.installer.yaml

```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.2.1 using InputObject 🤖 $debug=QUSU.CRLF.7-2-9
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: PrismLauncher.PrismLauncher
+PackageVersion: "6.3"
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-02-03
+Installers:
```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Reply:
The package installs, but requires the dependency to run:
![image](https://user-images.githubusercontent.com/114952565/218556886-eca84461-99e7-44f4-a929-0b514ef5014b.png)